### PR TITLE
updating governance for Velero - WIP

### PR DESCRIPTION
**Pull request by michmike**
Signed-off-by: Michael Michael <michmike@cs.stanford.edu>

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+| Maintainer | GitHub ID | Affiliation |
+| --------------- | --------- | ----------- |
+| Steve Kriss | [skriss](https://github.com/skriss) | [VMware](https://www.github.com/vmware/) |
+| Carlisia Campos | [carlisia](https://github.com/carlisia) | [VMware](https://www.github.com/vmware/) |
+| Nolan Brubaker | [nrb](https://github.com/nrb) | [VMware](https://www.github.com/vmware/) |
+| Ashish Amarnath | [ashish-amarnath](https://github.com/ashish-amarnath) | [VMware](https://www.github.com/vmware/) |
+| Stephanie Bauman | [stephbman](https://github.com/stephbman) | [VMware](https://www.github.com/vmware/) |
+
+## Emeritus Maintainers
```
[the remark this answers is not in the document]
Comment (michmike) [reply MAINTAINERS.md:15]: defined in governance

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Velero Core Contributors & Stakeholders
```
[the remark this answers is not in the document]
Comment (michmike) [reply MAINTAINERS.md:17]: defined in governance

File: MAINTAINERS.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (carlisia) [reply MAINTAINERS.md:15]: Hm. I did a search and there's only this one occurrence of the word "Emeritus". Did I miss it?

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+* **[Velero Plugin for vSphere](https://github.com/vmware-tanzu/velero-plugin-for-vsphere):** This repository contains the Velero Plugin for vSphere. This plugin is a volume snapshotter plugin that provides crash-consistent snapshots of vSphere block volumes and backup of volume data into S3 compatible storage.
+* **[Velero Plugin for AWS](https://github.com/vmware-tanzu/velero-plugin-for-aws):** This repository contains the plugins to support running Velero on AWS, including the object store plugin and the volume snapshotter plugin
+* **[Velero Plugin for GCP](https://github.com/vmware-tanzu/velero-plugin-for-gcp):** This repository contains the plugins to support running Velero on GCP, including the object store plugin and the volume snapshotter plugin
+* **[Velero Plugin for Azure](https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure):** This repository contains the plugins to support running Velero on Azure, including the object store plugin and the volume snapshotter plugin
+* **[Velero Plugin for AWS](https://github.com/vmware-tanzu/helm-charts/tree/master/charts/velero):** The Helm chart for the Velero server component
```
Comment (ashish-amarnath) [inline GOVERNANCE.md:20]: This line can be removed. "Velero Plugin for AWS" was covered on line 17 and also the link and description point to the Helm chart repo that was covered on line 14

File: MAINTAINERS.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+| Contour Feature Area | Lead |
```
Comment (ashish-amarnath) [inline MAINTAINERS.md:19]:
```suggestion
| Velero Feature Area | Lead |
```

Comment (ashish-amarnath) [inline MAINTAINERS.md:19]: Velero

File: MAINTAINERS.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (michmike) [reply MAINTAINERS.md:15]: oh you mean emeritus. that's a standard term for maintainers  that are retired from a project.

File: MAINTAINERS.md (hunk 4 above)
Comment (michmike) [reply MAINTAINERS.md]: good catch.fixed

File: GOVERNANCE.md (hunk 3 above)
Comment (michmike) [reply GOVERNANCE.md]: good catch. fixed.

File: design/_template.md [hunk 5]
```diff
@@ -1,35 +1,37 @@
-# Design proposal template (replace with your proposal's title)
+# Design proposal template `<replace with your proposal's title>`
 
-One to two sentences that describes the goal of this proposal.
-The reader should be able to tell by the title, and the opening paragraph, if this document is relevant to them.
+Status: {Draft,Reviewing,Accepted,Declined}
```
Comment (skriss) [inline design/_template.md:3]: per prev comment, I'd prefer not to add this back to our template.

File: MAINTAINERS.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+## Velero Contributors & Stakeholders
+
+| Feature Area | Lead |
+| ----------------------------- | :---------------------: |
+| Technical Lead | Steve Kriss (skriss) |
+| Kubernetes CSI Liaison | Nolan Brubaker (nrb), Ashish Amarnath (ashish-amarnath) |
+| Deployment | Carlisia Campos (carlisia), Carlos Tadeu Panato Junior (cpanato) | 
+| Community Management | Jonas Rosland (jonasrosland) |
+| Product Management | Stephanie Bauman (stephbman) |
```
Comment (kmova) [inline MAINTAINERS.md:25]: Would it make sense to have additional feature areas like - release management, plugins, documentation/website?

File: GOVERNANCE.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+* **[Velero Plugin for Azure](https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure):** This repository contains the plugins to support running Velero on Azure, including the object store plugin and the volume snapshotter plugin
+* **[Velero Plugin Example](https://github.com/vmware-tanzu/velero-plugin-example):** This repository contains example plugins for Velero
+
+
+## Community Roles
+
+* **Users:** Members that engage with the Velero community via any medium (Slack, GitHub, mailing lists, etc.).
+* **Contributors:** Regular contributions to projects (documentation, code reviews, responding to issues, participation in proposal discussions, contributing code, etc.). 
+* **Maintainers**: The Velero project leaders. They are responsible for the overall health and direction of the project; final reviewers of PRs and responsible for releases. Some Maintainers are responsible for one or more components within a project, acting as technical leads for that component. Maintainers are expected to contribute code and documentation, review PRs including ensuring quality of code, triage issues, proactively fix bugs, and perform maintenance tasks for these components.
```
Comment (kmova) [inline GOVERNANCE.md:27]: Should technical/feature leads be another role, a step forward from Contributors towards becoming a maintainer?

Comment (nrb) [reply GOVERNANCE.md:27]: Can you be more specific as to what makes that role more distinct from a contributor or maintainer?

File: MAINTAINERS.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (michmike) [reply MAINTAINERS.md:15]: @skriss add Andy as emeritus? yeah for sure. I didn't know he worked on this. Adnan is another option to add here if we want

File: MAINTAINERS.md (hunk 6 above)
Comment (michmike) [reply MAINTAINERS.md:25]: @kmova i am open to that once someone takes the lead for those areas. Meaning, if someone is willing to do that work and act as a lead in those areas, we can add that individual here too

File: design/_template.md (hunk 5 above)
Comment (michmike) [reply design/_template.md:3]: done.

File: GOVERNANCE.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+
+### Maintainers
+
+New maintainers must be nominated by an existing maintainer and must be elected by a supermajority of existing maintainers. Likewise, maintainers can be removed by a supermajority of the existing maintainers or can resign by notifying one of the maintainers.
+
+### Supermajority
+
+A supermajority is defined as two-thirds of members in the group.
```
[the remark this answers is not in the document]
Comment (michmike) [reply GOVERNANCE.md:35]: it rounds up. that's how supermajority works for two-thirds. essentially the folks `for` should be twice as many or more versus `against`

File: GOVERNANCE.md [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+
+The proposal PR can be marked with different status labels to represent the
+status of the proposal:
+
+* **Draft**: Proposal is just created.
+* **Reviewing**: Proposal is under review and discussion.
+* **Accepted**: Proposal has been reviewed and is accepted (either by consensus or through a vote).
+* **Declined**: Proposal has been reviewed and was rejected (either by consensus or through a vote).
+
+## Lazy Consensus
```
[the remark this answers is not in the document]
Comment (michmike) [reply GOVERNANCE.md:85]: lazy consensus is the practice of delivering an SLA for responding to proposals. ideas, by other maintainers etc.
supermajority is strictly used for decision making by the maintainer team

lazy consensus guards against a maintainer not being able to work on the project any more or dedicate time to review/comment on proposals

File: GOVERNANCE.md (hunk 7 above)
Comment (kmova) [reply GOVERNANCE.md:27]: It is like giving a category to the Feature leads (Stake holders) that are mentioned here: https://github.com/vmware-tanzu/velero/blob/a86079220173c4e7f5f9dead9aa5857ba77ba758/MAINTAINERS.md#velero-contributors--stakeholders

The journey could be: 
Contributors -> Leads/Stake holders -> Maintainers

As the load/backlog on the Maintainers increases, Maintainers could nominate one of the Contributors who are familiar with a certain feature/area and have significantly contributed to that area to become a Lead/Stake Holder

Comment (michmike) [reply GOVERNANCE.md:27]: @kmova that's a good suggestion. others what do you think?
cc: @vmware-tanzu/velero-maintainers

Comment (carlisia) [reply GOVERNANCE.md:27]: I think this is worth talking about "face-to-face" in our community meeting. 

Initially my question is, other than on-paper recognition (which I'm all for), how would this role be different than a maintainer, in practical terms?

**Pull request merged**